script/Counter.s.sol

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.9;
+
+import "forge-std/Script.sol";
+
+contract CounterScript is Script {
```

Comment:
This should be under the `/test` folder

Reply:
Foundry by default uses the `/script` folder for [solidity scripting](https://book.getfoundry.sh/tutorials/solidity-scripting) hence the `.s.sol` extension and not `.t.sol`, so I don't think it would make sense to move it into `./test/foundry`.

Reply:
Okay that's fine. `./scripts` folder makes sense.
Also looking at OZ's repo and seeing a similar approach.
Thanks

Reply:
OZ's `./scripts` folder doesn't seem to have any solidity scripts, only [hardhat js scripts](https://hardhat.org/hardhat-runner/docs/advanced/scripts)